### ISSUE TITLE
[Feature request] Add support for SmartCool AX206 variant

### ISSUE THREAD

**Issue by ImAleeexx** [SHWotever/SimHub#1424]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
A implementation of SmartCool display like an AX206 display

**Describe alternatives you've considered**

**Additional context**
Some displays on AliExpress are AX206 based but have different VID and PID like this SmartCool display.

I managed to self sign the AX206 driver but I cannot test if it works as the VID and PID of the display are USB\VID_2A55&PID_9990&REV_0200

I dont know if it uses same commands to communicate via scsi but it would be great to see this screen working on SimHub
If you need any more info just ask

**Comment by SHWotever** [comment]
Hi ! do you have a link for the product on aliexpress ? If it's a complete different variant (the different pid/vid tends to prove it) I would like to get a sample to check it "physically" (the AX206 already has some bad firmware variations, and before introducing an "unkown" variant I wan't to be able to check this variation myself).

**Comment by ImAleeexx** [comment]
Sure, here are some listings 
https://es.aliexpress.com/item/1005005902857202.html

**Comment by Mystereon** [comment]
I've got one of these screens and they are a huge pain in the ass to get working . I've made a repo of 480x320 backgrounds .https://github.com/Mystereon/SMARTCOOL-GUI_concepts_for_the-_AX206_480x320
Please get this working on SimHub , I normally code for esp32 and 8266 , to be fair I could get  new library put together if i have a roadmap so to speak .  Keenly watching this :D

**Comment by SHWotever** [comment]
I'm still waiting for a test device, but I got some additional informations about this "smartcool" ax206 (fake chip/firmware??) LCD with the help of another user. 

His variation had the correct pid/vid, but was crashing as soon as the picture was sent. After a bit of investigation we found that it was not supporting partial draws which are required to leverage a decent frame rate. Without those partial draws it falls to a rude "2fps" but works ... for the next version I added an option to be able to disable it, it's slow but it's the best those variations can do unfortunately.

I know that aida64 does not focus on screen refresh rate and only do full screen refreshes one or two times per second at best. That could explain why those variations went through, despite such firmware bug/lack (like we say "works in my computer" or more precisely "works on aida64" in this case). I can't tell if it's a fake chip, if the firmware has been stripped down to remove anything useless for aida64, or both ... I hope I will be able to tell a little more about it when I will be able to tear down this variation.

**Comment by ImAleeexx** [comment]
Is there any way to reflash the AX206 chip with other firmware?

**Comment by SHWotever** [comment]
https://github.com/dreamlayers/dpf-ax
It's the official firmware for usb control, I never went through trying to update it myself. At your own risks ;)

**Comment by SHWotever** [comment]
I received this smart cool thing ... I confirm it's somehow a different firmware, aida64 shows some additional settings like compression quality, but I could not find any documentations, and even with that and compression cranked to the lowest aida does around 0.5PFS/1FPS (From what I could measure visually).

It's somehow compatible with the AX206 protocol and with simhub it goes around 2FPS (Simhub tries to feed it as fast as possible, while I suspect AIDA to just have a fixed delay between two refreshes).

On the good side it's working "driver less" with a proper winusb implementation.

I added it with correct PID/VID and forced disabled partial draws for it but it's really not doing miracles :(
For comparaison, top is a true AX206, bottom is this smartcool :
https://www.youtube.com/watch?v=U19ELsCJD_M

True AX206 does the trick so much better ...

I took the opportunity to refresh the documentation with these new informations : https://github.com/SHWotever/SimHub/wiki/Dashstudio---USB-Displays#ax206-warning

AX206 stayed a mystery for long, I bought many of thems hoping to find a "non working" one , but I guess fake firmware missing the partial draws have been a thing for long, maybe to save space for idle picture or other options.

I close this, the support will be included in the next version ;)
